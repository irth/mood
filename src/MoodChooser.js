import React from 'react';

import MoodEmoji from './MoodEmoji';

const EmojiButton = ({onClick, mood}) => (
  <a onClick={() => onClick(mood)} className="bigEmoji">
    <MoodEmoji mood={mood} size={50} />
  </a>
);

export default ({onSelected}) => (
  <div className="Chooser">
    <EmojiButton onClick={onSelected} mood={1} />
    <EmojiButton onClick={onSelected} mood={2} />
    <EmojiButton onClick={onSelected} mood={3} />
    <EmojiButton onClick={onSelected} mood={4} />
    <EmojiButton onClick={onSelected} mood={5} />
  </div>
);
