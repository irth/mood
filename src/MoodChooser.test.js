import React from 'react';
import MoodChooser from './MoodChooser';
import renderer from 'react-test-renderer';

it('matches the snapshot', () => {
  const component = renderer.create(<MoodChooser />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
