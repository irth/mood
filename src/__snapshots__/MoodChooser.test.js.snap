// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="Chooser"
>
  <a
    className="bigEmoji"
    onClick={[Function]}
  >
    <span
      className="undefined"
      style={
        Object {
          "fontSize": 50,
        }
      }
    >
      <img
        alt="😶"
        className={undefined}
        src="http://twemoji.maxcdn.com/2/72x72/1f636.png"
        style={
          Object {
            "height": "1em",
            "margin": "0 .05em 0 .1em",
            "verticalAlign": "-0.1em",
            "width": "1em",
          }
        }
      />
    </span>
  </a>
  <a
    className="bigEmoji"
    onClick={[Function]}
  >
    <span
      className="undefined"
      style={
        Object {
          "fontSize": 50,
        }
      }
    >
      <img
        alt="😞"
        className={undefined}
        src="http://twemoji.maxcdn.com/2/72x72/1f61e.png"
        style={
          Object {
            "height": "1em",
            "margin": "0 .05em 0 .1em",
            "verticalAlign": "-0.1em",
            "width": "1em",
          }
        }
      />
    </span>
  </a>
  <a
    className="bigEmoji"
    onClick={[Function]}
  >
    <span
      className="undefined"
      style={
        Object {
          "fontSize": 50,
        }
      }
    >
      <img
        alt="😐"
        className={undefined}
        src="http://twemoji.maxcdn.com/2/72x72/1f610.png"
        style={
          Object {
            "height": "1em",
            "margin": "0 .05em 0 .1em",
            "verticalAlign": "-0.1em",
            "width": "1em",
          }
        }
      />
    </span>
  </a>
  <a
    className="bigEmoji"
    onClick={[Function]}
  >
    <span
      className="undefined"
      style={
        Object {
          "fontSize": 50,
        }
      }
    >
      <img
        alt="😊"
        className={undefined}
        src="http://twemoji.maxcdn.com/2/72x72/1f60a.png"
        style={
          Object {
            "height": "1em",
            "margin": "0 .05em 0 .1em",
            "verticalAlign": "-0.1em",
            "width": "1em",
          }
        }
      />
    </span>
  </a>
  <a
    className="bigEmoji"
    onClick={[Function]}
  >
    <span
      className="undefined"
      style={
        Object {
          "fontSize": 50,
        }
      }
    >
      <img
        alt="😄"
        className={undefined}
        src="http://twemoji.maxcdn.com/2/72x72/1f604.png"
        style={
          Object {
            "height": "1em",
            "margin": "0 .05em 0 .1em",
            "verticalAlign": "-0.1em",
            "width": "1em",
          }
        }
      />
    </span>
  </a>
</div>
`;
